src/utils/style-formatter.js

```diff
@@ -79,15 +84,29 @@ export default {
   },
   setFontSizeAndFamily({ button, Theme, layout }) {
```

Comment:
As you mentioned, this function is a bit messy. Off the top of my head, can't think of a smarter way to do it. So I guess we can improve the readability at least. This function needs to be renamed for what it is actually doing, `createLabelAndIcon` or something.